### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone. You should also not use `ldd` to copy dependencies.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/sdl-mixer-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Omit image name; tests -> testsuite.

File: fuzz/fuzz.c [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#include <stdlib.h>
```
Comment (xansec) [inline fuzz/fuzz.c]: Can we rename to indicate what is being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec I have made these changes, please let me know if anything else is needed

**Comment by xansec** [comment]
Looks like there's an upstream conflict. Can you sync with upstream first?

**Comment by capuanob** [comment]
@xansec Synced!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**